### PR TITLE
Feature/member controller - finished Member Controller part.

### PR DESCRIPTION
**Pull request by Derasm**
Finished the member controller API, Server and everything.Let me know if anything needs changing, so i can dutifully do it or ignore it, depending on the hilarity of the impending action ;)

### CODE REVIEW

**Review by Federlizer — APPROVED** [review]
Looks good to me. Only thing to note is that on the Api.Controller, the PostMember method accepts all those parameters through the URL (for example, POST http://localhost:5000/api/member?SSN=ssn&fName=fname&lName=lname..........), which is valid. However so far all the controllers have been using JSON bodies to accept the data as an Object. You can do something like this:

```cs
public Member PostMember([FromBody] Api.Models.Member member) {
    // inside you'll have the built member type,
    // however, the Api.Models one not the Server.Models one.
}
```

This to me seems like an easier solution rather than having to have all the parameters listed out on the method.

**Comment by ignaczalexander** [comment]
I would also suggest to change the PostMember method, its generally a best practice to use body parameters for POST methods. Also, the Delete method accepts a Member as a parameter which is also fine but it would make it simpler for the client if you just ask for an ID, we did it like that for the rest of the API routes.

File: GeorgiaTech/Api/Models/Member.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        }
+
+        public int MemberId { get; set; }
+        public string SSN { get; set; }
+        public string FName { get; set; }
+        public string LName { get; set; }
+        
+
+        public virtual ICollection<string> PhoneNumbers { get; set; }
+        public virtual string CampusAddress { get; set; }
```
Comment (ignaczalexander) [inline GeorgiaTech/Api/Models/Member.cs:18]: You could also just reference the `Api.Models.Address` class for both CampusAddress and HomeAddress.

**Review by ignaczalexander — COMMENTED** [review]
I can't seem to call the API endpoints, get the following exception:
```
Microsoft.AspNetCore.Routing.Matching.AmbiguousMatchException: The request matched multiple endpoints. Matches: 

Api.Controllers.MemberController.PostMember (Api)
Api.Controllers.MemberController.DeleteMember (Api)
Api.Controllers.MemberController.UpdateMember (Api)
Api.Controllers.MemberController.GetMemberByID (Api)
Api.Controllers.MemberController.GetMemberByName (Api)
Api.Controllers.MemberController.FindAllByType (Api)
```

File: GeorgiaTech/Api/Models/Member.cs (hunk 1 above)
Comment (Derasm) [reply GeorgiaTech/Api/Models/Member.cs:18]: Gonna refactor it to that then - no reason not to use them. 
- Gonna clean up the API endpoints so it doesn't conflict.

**Pull request merged**